lib/resources/r4/encounter.yaml

```diff
@@ -573,12 +573,40 @@ fields:
         }
       ],
       "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+    },
+    {
+      "extension": [
```

Comment:
nit: single quotes

Reply:
It looks like we use double quotes throughout this file and other resource yaml files.